pages/user guides/discover_nmap.md

```diff
@@ -4,6 +4,7 @@ keywords: nmap, ip, address, api, services
 last_updated: September 2020
 summary: "How to use nmap to scan your network"
 sidebar: main_sidebar
+folder: userguides
```

Comment:
curious for myself, what does this do? not sure I've used or seen it before

Reply:
I'm a bit curious myself. I copied the format from Public Urls.  What is interesting is that not all the files inside the folder `userguides` use it.  some do, but others don't. I think it might help Jekyll find files with the same name in different folders by allowing us to specify which folder is the correct one. 

![image](https://user-images.githubusercontent.com/22829270/111635096-e84bd600-87d5-11eb-943f-c24128c4babd.png)